docs/course_materials/variables_data_types.html

```diff
@@ -39,12 +39,75 @@
             <section data-background-size="cover" data-background-image="media/rose_background.jpg">
                 <h1 class="slide_main_title"><b>Variables and Data Types</b></h1>
             </section>
+            <section>
+                <section>
+                    <h1><b>
+                            <span>Interactive Input and String Formatting</span>
+                    </h1></b>
+                </section>
+                <section>
+                    <h3><b>Interactive Input</b></h3>
+                    <ul>
+                        <li>In order to get an input:</li>
+                        <ul>
+                            <li>input()</li>
+                            <li>input(prompt) : prints prompt before reading input</li>
+                        </ul>
+                    </ul>
+                    <pre><code data-trim data-noescape class="hljs">
+>>> name = input('Enter your name: ')
+Enter your name: 'John Snow'
+>>> print(name)
+John Snow
+
+>>> address = input('Enter your address: ')
+Enter your address: 'Tel Aviv'
+>>> print(address)
+Tel Aviv
+                    </code></pre>
+                </section>
+                <section>
+                    <h3><b>Formatting</b></h3>
+                    <pre><code data-trim data-noescape class="hljs">
+    >>> name = 'Tom'
+    >>> family_name = 'Sawyer'
+    >>> print('Hello %s!' % name)
+    >>> Hello Tom!
+    >>> print('Hello %s %s!' % (name, family_name))
+    >>> Hello Tom Sawyer!
+                    </code></pre>
+                </section>
+                <section>
+                <section data-markdown="exercises/03_Variables_and_datatypes/class_exercise_1.md" data-separator="^\n\n\n" data-separator-vertical="^\n\n">
+                </section>
+            </section>
+            </section>
```

Comment:
Is this one redundant too?
Please make sure you can see the main rose background on the first presentation slide